cli/admin/src/main/java/io/pravega/cli/admin/AdminCommand.java

```diff
@@ -472,6 +478,13 @@ private String objectToJSON(Object object) {
         return new ObjectMapper().writeValueAsString(object);
     }
 
+    protected Controller instantiateController() {
+        ConnectionPool pool = createConnectionPool();
```

Comment:
connection pool will not be closed when Controller.close() is invoked.

Reply:
done